cli/src/main/java/bisq/cli/TradeFormat.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.cli;
+
+import bisq.proto.grpc.TradeInfo;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import java.util.function.Supplier;
+
+import static bisq.cli.ColumnHeaderConstants.*;
+import static bisq.cli.CurrencyFormat.formatOfferPrice;
+import static bisq.cli.CurrencyFormat.formatSatoshis;
+import static com.google.common.base.Strings.padEnd;
+
+@VisibleForTesting
+public class TradeFormat {
+
+    @VisibleForTesting
+    public static String format(TradeInfo tradeInfo) {
+        // Some column values might be longer than header, so we need to calculated them.
```

Comment:
```suggestion
        // Some column values might be longer than header, so we need to calculate them.
```

Reply:
Resolved in PR https://github.com/bisq-network/bisq/pull/4731, commit https://github.com/bisq-network/bisq/pull/4731/commits/3cd3bf0e3ed3aed626fe38ef1b1da29cbc761c61.